apps/src/netsim/NetSimDnsTable.html.ejs

```diff
@@ -0,0 +1,44 @@
+<%
```

Comment:
D'oh.  I should be.  I'll go convert this.  Do I have to change anything about how it's called?